src/app/pages/PupilOverview/PupilOverview.tsx

```diff
@@ -19,7 +20,7 @@ export default function PupilOverview(): JSX.Element {
 
   function handleFormSubmit(category: string, evaluation: string) {
     addEvaluation(currentPupil, category, evaluation);
-    setIsFormShown(false);
+    setIsAddEvaluationFormShown(false);
```

Comment:
I would also consider something like AddEvaluationFormShowStatus or AddEvaluationFormShowIndictor, but that's all down to personal preferences is guess :)